test/lambda/mongodb/package.json

```diff
@@ -5,6 +5,6 @@
   "main": "app.mjs",
   "license": "MIT",
   "dependencies": {
-    "mongodb": "^5.0.1"
+    "mongodb": "mongodb/node-mongodb-native#NODE-5197-server-monitoring-mode"
```

Comment:
This will fail when we merge this and GH auto deletes this branch. Since we've observed it run green, do we want to change it back here? Also, it seems wrong that this was installing 5.x, can we specify a file directory like ../.. so it uses the current driver? (we can defer that if there are setup issues)

Reply:
Yes I'll change this back to main. Pointing it to a file fails on `sam build` as it somehow gets into an infinite loop doing the npm install, so I needed to go to Github instead.

Reply:
Updated to use main.